docs/Configuration/Overview.md

```diff
@@ -15,30 +15,54 @@ variables][link-env-vars].
 
 ## Configuration Precedence
 
-In Cube.js, environment variables take precedence over values specified
-in `cube.js`. In production scenarios, it allows changing configuration
-by simply switching environment variables, which is generally safer,
-faster and easier than re-deploying the codebase.
+In Cube.js, environment variables take precedence over values specified in
+`cube.js`. In production scenarios, it allows changing configuration by simply
```

Comment:
```suggestion
`cube.js`. In production scenarios, it allows changing configuration by
```

How simple it is, depends on the tools used, right? But won't it be more concise to omit "simply" here?